.env.production

```diff
@@ -1,2 +1,3 @@
 VITE_RC_ENDPOINT=https://api.revenuecat.com
+VITE_RC_ANALYTICS_ENDPOINT=https://api.mimircat.com
```

Comment:
FWIW, in mobile, we use a different subdomain for these events (`https://api-paywalls.revenuecat.com/` I know it says paywalls but it's used for all events right now)... But it makes more sense to have an entire separate domain 👍

Reply:
Thanks, good to know. I will check with coin.